### PR TITLE
PL-2030 Add new rule for sanitizer for better development

### PR DESCRIPTION
**Pull request by VlVl**
Added new function for flexible sanitizing of jsonb. 
Blocked by migration to update comments for sanitizing

link to jira card https://plyolabs.atlassian.net/browse/PL-2030
plyo.db PR https://github.com/plyo/plyo.db/pull/1704

### CODE REVIEW

**Comment by jifeon** [comment]
No link to issue, no link to blocking PR... please pay more attention for PR description

File: publisher/sql/sanitize.sql [hunk 1]
```diff
@@ -86,6 +86,49 @@ end
 $$
 language plpgsql;
 
+create or replace function sanitizing.get_sanitized_jsonb(j jsonb)
+  returns jsonb as
+$$
+declare
+  k text;
+begin
+  for k in select k1 from jsonb_object_keys(j) t(k1) loop
```
Comment (jifeon) [inline publisher/sql/sanitize.sql]: k, k1, j, t - how to read it?

File: publisher/sql/sanitize.sql [hunk 2]
```diff
@@ -86,6 +86,49 @@ end
...
+      if jsonb_typeof(j -> k) = 'string' then
+        if j ->> k ~ '.+@.+\..+' then
+          j = jsonb_set(j, array[k], to_jsonb((select
+                                                 string_agg(
```
Comment (jifeon) [inline publisher/sql/sanitize.sql]: please format it

File: publisher/sql/sanitize.sql [hunk 3]
```diff
@@ -152,5 +195,16 @@ from (select
                                                     and c.table_schema = st.schemaname and c.table_name = st.relname)
       where pgd.description ilike '%SANITIZE_AS_JSONB%') as res;
 
+-- call radically_sanitize_jsonb on every column containing SANITIZE_AS_EMPTY_JSONB in the comment
+select sanitizing.radically_sanitize_jsonb(res.table_name :: varchar, res.column_name :: varchar)
```
Comment (jifeon) [inline publisher/sql/sanitize.sql]: can we call it similarly to comment, like `sanitize_as_empty_jsonb`?

Comment (VlVl) [reply publisher/sql/sanitize.sql]: done

**Comment by VlVl** [comment]
@jifeon there is a problem - recursive updating of json takes about 2m (on my machine)

File: publisher/sql/sanitize.sql (hunk 3 above)
Comment (jifeon) [reply publisher/sql/sanitize.sql]: not pushed?

**Comment by jifeon** [comment]
need to deploy together with 234 version of web

**Pull request merged**